### PR TITLE
Official titles for Change Position

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

Adds a file that is barebones and only meant for changing positions of others to this unique, one of a kind sort of job.
The list currently is as follows:

Warden of the Terrorbog
Huntmaster of the Murderwoods
Hedtman of Mount Decapitation

And anything else you people can think of, I am open to suggestions.

## Why It's Good For The Game

More RP potential from the Ruler to their subjects.

image

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by messagecypher** [comment]
add nobility trait when u give the thing too :)

**Comment by ThePainkiller** [comment]
> add nobility trait when u give the thing too :)

Killing you with rocks

File: code/modules/jobs/job_types/roguetown/nobility/titles.dm [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	name = "Huntmaster of the Murderwoods"
+	jobtype = /datum/job/roguetown/huntmaster
+
+	pants = /obj/item/clothing/under/roguetown/tights
+	shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt
+	armor = /obj/item/clothing/suit/roguetown/shirt/rags
+	shoes = /obj/item/clothing/shoes/roguetown/boots
+
+/datum/job/roguetown/hedtman
+	title = "Hedtman of Mount Decapitation"
```
Comment (NPC1314) [inline code/modules/jobs/job_types/roguetown/nobility/titles.dm:46]: I think Hetman would be better

**Comment by Ickysoup** [comment]
Concubine title

**Comment by guayitoo** [comment]
Thank you for finally adding this! Much appreciate <3

- Guayo

**Comment by NPC1314** [comment]
......Hetman isn't english you absolute tool, hetwoman sounds so stupid I feel like sandpapering my brain.

**Comment by BadAtThisGame302** [comment]
> ......Hetman isn't english. Hetwoman sounds so stupid I feel like sandpapering my brain. Its like someone changing the language from Mandarin to Womandorin when a woman speaks it.
> 
> Just delete the title at that point, I cant take the suffering

image

**Review by NPC1314 — APPROVED** [review]
No reports of any bugs and seems like a good addition.

**Pull request merged**